BEASTIE/Get_LD.R

```diff
@@ -59,7 +59,7 @@ Get_LD <- function(snp,lag_snp,mytoken,pop,r2d){
 }
 
 loop_chr <-function(outpath,data,ancestry,batch_size,mytoken,chr_start,chr_end){
-  final_df_filename=paste0(outpath,"/",prefix,"_chr",chr_start,"-",chr_end,"_LD.TEMP.tsv",sep="")
+  final_df_filename=paste0(outpath,"/TEMP.",prefix,"_chr",chr_start,"-",chr_end,"_LD.tsv",sep="")
```

Comment:
should this be tmppath?

Reply:
These files are in tmp path, I just changed the filename to starts with "TEMP". so that I could removed some tmp files generated through annotation LD process